tests/nndc_test.py

```diff
@@ -31,114 +27,114 @@ class TestParseFloatUncertainty(object):
 
     def test_01(self):
         """Test _parse_float_uncertainty('257.123', '0.005')..............."""
```

Comment:
are all the `.......` necessary?

Reply:
Haha, no, that is a very old hack from me to get the output from `nose` to look pretty and aligned when there are a lot of tests. Not part of this PR, but can be removed in the future.